lib/net.js

```diff
@@ -64,15 +64,16 @@ exports.connect = exports.createConnection = function() {
   var args = new Array(argsLen);
   for (var i = 0; i < argsLen; i++)
     args[i] = arguments[i];
-  args = normalizeConnectArgs(args);
+  args = normalizeArgs(args);
   debug('createConnection', args);
   var s = new Socket(args[0]);
   return Socket.prototype.connect.apply(s, args);
 };
 
-// Returns an array [options] or [options, cb]
+// Returns an array [options, cb], where cb can be null.
 // It is the same as the argument of Socket.prototype.connect().
-function normalizeConnectArgs(args) {
+// This is also used by Server.prototype.listen().
```

Comment:
This should probably say that it is used by `listen()` and `connect()`. Just saying "also" doesn't mean much when the "connect" part is removed.